backend/api/models/PortfolioManager.py

```diff
@@ -13,13 +13,11 @@ class PortfolioManager(Mixin, db.Model):
     id = db.Column(db.String, unique=True, primary_key=True)
     email = db.Column(db.String)
     name = db.Column(db.String)
-    list_of_fps = db.Column(ARRAY(db.String), nullable=True)
 
-    def __init__(self, email, name, list_of_fps):
+    def __init__(self, data):
         self.id = "p" + str(uuid.uuid4())
-        self.email = email
-        self.name = name
-        self.list_of_fps = list_of_fps

```

Comment:
We shoudn't be storing things in arrays with Postgres, and since we already store a Porfolio Manager ID with each Field Partner, we can simply query the Field Partner table to find all the Field Partners for a given Portfolio Manager.